pq-interop/pq-devnet-0.md

```diff
@@ -0,0 +1,137 @@
+
+
+### Test Releases
+
+**Consensus Specs:** link to consensus spec release (Felipe from STEEL will set this up the week of July 28)
+**Execution Spec Tests:** Not applicable
+
+### Spec Versions Required & Open PRs
+
+- List of links to spec PR's **Merged** :heavy_check_mark: 
+- List of links to spec PR's **Open** :exclamation:
+
+
+### Client Support Status
+>develop list of testing scenarios (link here)
+
+| Testing Scenario | Ream | Zeam | [TBD: Third Client?] |
+|------------------|------|------|----------------------|
+| 1a | :question: | :question: | :question: |
+| 2a | :question: | :question: | :question: |
+| 3a | :question: | :question: | :question: |
+| 4a | :question: | :question: | :question: |
+| 5a | :question: | :question: | :question: |
+| 6a | :question: | :question: | :question: |
+
+## Kurtosis Interop Config (Pre-Devnet Testing)
+
+>yaml below is AI slop
+
+```yaml
+participants_matrix:
+  cl:
+    - cl_type: ream
+      cl_image: reamlabs/ream:pq-devnet-0
+    - cl_type: zeam
+      cl_image: zeam/zeam:pq-devnet-0
+    - cl_type: [TBD - third client, e.g., lighthouse-pq]
+      cl_image: [TBD]:pq-devnet-0
+
+network_params:
+  max_validators: 1000  # Initial keys, scale to 10k+
+  slot_time_seconds: [TBD - 4 or 12]
+
+additional_services:
+  - metrics_dashboard  # Shared Prometheus/Grafana
+  - genesis_generator
+
+metrics_params:
+  image: [TBD - ethpandaops/metrics:master]
+  dashboards:
+    - scenario: pq_signing
+      config:
+        throughput: 100  # Signatures per slot
+    - scenario: propagation
+      config:
+        throughput: 50  # Blocks/attestations
+```
+
+### Testing:
+
+>testing ideas below is AI slop
+
+#### PQ Signature Viability
+  * Key generation and signing/verification timings - kurtosis - generate keys for 1,000 validators and measure sub-second targets.
+  * Adversarial testing: Inject bad signatures via custom actors; verify rejection.
+  * EIP-7870 compliance: Monitor CPU/memory/bandwidth during runs.
+
+#### Multi-Client Interop
+  * Block-by-root: Use for recovery to catch-up to chain head when nodes break; include chianing to pull parent chains for latest gossip blocks.
+  * Attestation correctness: Assertoor-like test for ≥98% (Zeam) or local tally (Ream).
+
+#### Resource Requirements
+  * Run with 1,000 validators (devnet-0); scale by 10x validators for each iteration to 10,000; measure EIP-7870 limits.
+  * Hardware metrics: CPU load, memory, throughput every second.
+  * Bonus: Inject adversaries (signature collisions, censoring); monitor resilience.
+
+#### Success Criteria Validation
+  * Network runs for at least 300 slots * num_clients without crashing for local testing; for cluster devnet, aim for longer durations (e.g., 10 days), subject to test infrastructure capabilities (Extended runs will be beneficial for incorporating features like 3SF in subsequent devnet iterations).
+  * Full block pipeline <6s (Ream) or on-time with 4s slots (Zeam).
```

Comment:
```suggestion
  * Full block pipeline within <4 seconds
```

Reply:
Maybe just `<4 seconds`? or `within 4 seconds`?

Reply:
updated to in <4 seconds since it gives more clarity than "within"